api/v1/reviews/PKPReviewController.php

```diff
@@ -131,7 +131,7 @@ public function getHistory(Request $illuminateRequest): JsonResponse
         $declineEmail = null;
         if ($reviewAssignment->getDeclined()) {
             $emailLogs = EmailLogEntry::withAssocId($submissionId)
-                ->withEventType(SubmissionEmailLogEventType::REVIEW_DECLINE)
+                ->withEventTypes([SubmissionEmailLogEventType::REVIEW_DECLINE])
```

Comment:
Just wanting to make sure you checked the OJS/OMP/OPS repos for any place this method appears to make sure it's changed there as well.

Reply:
Yes, I did check. This is the only place where the method is being used.